libs/agora/ui/src/lib/footer/footer.component.html

```diff
@@ -1,39 +1,23 @@
-<footer class="mat-typography">
-  <div class="row">
-    <div class="col-8 about-oc">
-      <img class="logo" src="/agora-assets/images/agora-color.svg" alt="Agora logo" />
-      <ul class="footer-link-group">
-        <li><a routerLink="/about">About</a></li>
-        <li><a routerLink="/team">Meet Our Team</a></li>
-        <li>
-          <a
-            [href]="apiDocsUrl"
-            title="Interactive documentation for OC API"
-            rel="noopener noreferrer"
-            target="_blank"
-            >API Docs</a
-          >
-        </li>
-        <li>
-          <a
-            href="https://github.com/Sage-Bionetworks/sage-monorepo/issues/new/choose"
-            title="Share feedback about the OC app"
-            rel="noopener noreferrer"
-            target="_blank"
-            >Submit a Feature/Bug</a
-          >
-        </li>
-      </ul>
-    </div>
-    <div class="col-4 app-info">
-      <ul>
-        <li>App version: {{ appVersion }}</li>
-      </ul>
-    </div>
+<footer>
+  <div class="footer-nav" *ngIf="navItems.length">
+    <ul>
+      <li class="footer-logo">
+        <a routerLink="/">
+          <img [src]="footerLogoPath" alt="footer logo" />
+        </a>
+      </li>
+      <li *ngFor="let item of navItems">
+        <a *ngIf="item.routerLink" [routerLink]="item.routerLink" routerLinkActive="active"
+          ><span>{{ item.label }}</span></a
+        >
+        <a *ngIf="item.url" [attr.href]="item.url" [attr.target]="item.target"
```

Comment:
The behavior is that if the link does not exist, then nothing is displayed, correct? Would it make sense to show the label even if the url is falsy?

Reply:
makes sense, i can implement that.